csrc/mmdeploy/core/tensor.h

```diff
@@ -14,11 +15,14 @@ namespace mmdeploy {
 namespace framework {
 
 using TensorShape = std::vector<int64_t>;
+using TensorStride = std::vector<int64_t>;
```

Comment:
Remove strides support and simply set `DLTensor::strides` to `nullptr`. Adding strides before we have a detailed plan to support strides is dangerous. As the rest of the codebase does not support strided tensors, `is_contiguous` must be checked everywhere. It also enlarges `Tensor` size and complicates tensor opreations.